py3status/modules/do_not_disturb.py

```diff
@@ -90,13 +90,14 @@ class Dunst(Notification):
     """
 
     def toggle(self, state):
```

Comment:
use `pkill -x` to have a less fragile experience 😉

Reply:
Using `-x` does not work in NixOS, probably for the same reason that `killall` does not work either.

Reply:
use `pkill '^dunst$'` to have a less fragile experience :wink:

Reply:
just out of curiosity, @m45t3r could you post the result of `ps aux | grep dunst`? Wondering how exactly the process is executed in NixOS.

Reply:
@lasers Also does not work

@maximbaz 
```
 ps aux | grep dunst
thiagoko  1588  0.0  0.0 553660 16932 ?        Sl   11:49   0:00 /nix/store/71wp6r2pj4sfgf061jfmzkr7jfffwj3l-dunst-1.3.2/bin/dunst
thiagoko  2227  0.0  0.0 133228  2296 pts/5    R+   11:49   0:00 grep --color=auto dunst
```
Where `/nix/store/71wp6r2pj4sfgf061jfmzkr7jfffwj3l-dunst-1.3.2/bin/dunst` is a script:
```
#! /nix/store/a9i0a06gcs8w9fj9nghsl0b6vvqpzpi4-bash-4.4-p23/bin/bash -e
export GDK_PIXBUF_MODULE_FILE='/nix/store/mfwiwwqvmsxqpmhvq61aqivvkaqzc1bm-librsvg-2.42.4/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache'
exec -a "$0" "/nix/store/71wp6r2pj4sfgf061jfmzkr7jfffwj3l-dunst-1.3.2/bin/.dunst-wrapped"  "${extraFlagsArray[@]}" "$@"
```

Reply:
Interesting, I don't really understand why `killall` or `pgrep -x` doesn't work for you, in my case it is `/usr/bin/dunst` and in your case it is `/nix/store/.../dunst`, but in both cases the executable name is `dunst`... 🤔 

I found [this thread](https://www.linuxquestions.org/questions/linux-general-1/killall-process-not-found-4175641286/), but it has no answer either.

Anyhow, I'm fine keeping `pkill dunst`, it's just my curiosity 🙂

Reply:
Huh... Probably because of this
```
ps -e -o fname,cmd,comm,args | grep "[d]unst"
.dunst-w /nix/store/71wp6r2pj4sfgf06 .dunst-wrapped  /nix/store/71wp6r2pj4sfgf061jfmzkr7jfffwj3l-dunst-1.3.2/bin/dunst
```
So the binary name is actually `.dunst-wrapped`, the reason why `-x` or `^dunst$` doesn't work.

Reply:
Interesting, now it at least makes sense 😄 

```
❯ ps -e -o fname,cmd,comm,args | grep "[d]unst"
dunst    /usr/bin/dunst              dunst           /usr/bin/dunst
```